.env

```diff
@@ -0,0 +1,6 @@
+SERVER_PORT = 4367
```

Comment:
why added env variables with credentials in git?
create `.env.sample` and just add this 
```
DB_HOST=
DB_USERNAME=
DB_PORT=
DB_DATABASE=
DB_PASSWORD=
```

also and `.env` in `.gitignore`